packages/esm-active-visits-app/src/visits-summary/visit-detail.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('VisitDetailComponent', () => {
     render(<VisitDetailComponent visitUuid={visitUuid} patientUuid={patientUuid} />);
 
     expect(screen.getByText(/Some Visit Type/)).toBeInTheDocument();
-    expect(screen.getByText(formatDate(visitDate))).toBeInTheDocument();
+    expect(screen.getByText(formatDate(visitDate), { collapseWhitespace: false })).toBeInTheDocument();
```

Comment:
Not sure why this was working for others, but I had to add this in, because the time component was getting rendered with a non-breaking space between the numeric time and the AM/PM. (I believe this is an underlying javascript library issue, not an issue with our code).   Looks like the testing framework collapses all whitespace of any type to a typical whitespace, and so therefore the the match wasn't being made.